src/Namshi/Innovate/Client.php

```diff
@@ -167,6 +181,35 @@ public function createMpiRequest($method = 'GET', $uri = null, $headers = null,
     }
 
     /**
+     * Given a cart id reference will search for transaction for that cart id on innovate
+     *
+     * @param $ref
+     * @return \SimpleXMLElement
+     * @throws InnovateException
+     */
+    public function searchPaymentsByCartId($ref)
```

Comment:
it's "by cartid" but in the innovate doc it is called ref. I was wondering what to do as well.
But given that we both have the same thought, I will go for $cartId ;)